Tests/NIOTests/ChannelTests.swift

```diff
@@ -2565,6 +2565,100 @@ public class ChannelTests: XCTestCase {
+                                                                                                  TCP_NODELAY)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server1.getOption(option: ChannelOptions.socket(SOL_SOCKET,
+                                                                                                  SO_KEEPALIVE)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server1.getOption(option: ChannelOptions.socket(IPPROTO_TCP,
+                                                                                                  TCP_NODELAY)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try client2.getOption(option: ChannelOptions.socket(SOL_SOCKET,
+                                                                                                  SO_REUSEADDR)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertEqual(try client2.getOption(option: ChannelOptions.socket(IPPROTO_TCP,
+                                                                                            TCP_NODELAY)).wait(),
+                                        0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server2.getOption(option: ChannelOptions.socket(SOL_SOCKET,
+                                                                                                  SO_KEEPALIVE)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server2.getOption(option: ChannelOptions.socket(IPPROTO_TCP,
+                                                                                                  TCP_NODELAY)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertEqual(try client3.getOption(option: ChannelOptions.socket(SOL_SOCKET,
+                                                                                            SO_REUSEADDR)).wait(),
+                                        0))
+        XCTAssertNoThrow(XCTAssertEqual(try client3.getOption(option: ChannelOptions.socket(IPPROTO_TCP,
+                                                                                            TCP_NODELAY)).wait(),
+                                        0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server3.getOption(option: ChannelOptions.socket(SOL_SOCKET,
+                                                                                                  SO_KEEPALIVE)).wait(),
+                                              0))
+        XCTAssertNoThrow(XCTAssertGreaterThan(try server3.getOption(option: ChannelOptions.socket(IPPROTO_TCP,
+                                                                                                  TCP_NODELAY)).wait(),
+                                              0))
```

Comment:
formatting looks really odd... maybe merge with previous line ? (same comment for others)

Reply:
@normanmaurer then the line would be too long. And in github you couldn't see any of the important bits as they were hidden off screen. Still want me to merge?

Reply:
maybe use this formatting then ?:

```swift
let value = XCTAssertNoThrow(try server3.getOption(option: ChannelOptions.socket(IPPROTO_TCP,TCP_NODELAY)).wait())
XCTAssertGreaterThan(value,  0))

```

Reply:
just put the shared code in a function actually